src/Update.cpp

```diff
@@ -1246,6 +1251,23 @@ void DoPlaceClose(int i, int j, unsigned short int ts, int tn, int DoAnyway)
 	}
 }
 
+void UpdateHostClosure() {
+    // Single-threaded here, because the same host index might come up in
+	// different threads. (Which is the point of this buffered update...)
```

Comment:
Whitespace is inconsistent here (looks like tabs vs spsaces)

Reply:
Done!